machine.go

```diff
@@ -578,7 +579,11 @@ func (m *Machine) StopVMM() error {
 func (m *Machine) stopVMM() error {
 	if m.cmd != nil && m.cmd.Process != nil {
```

Comment:
Out of curiosity, could we use [`ProcessState`](https://golang.org/pkg/os/#ProcessState) to check whether or not the process has exited?

Reply:
I think, the process might be finished between checking (`m.cmd.ProcessState.Exited()`) and signaling (`m.cmd.Process.Signal(syscall.SIGTERM)`).

Reply:
Ah, true there is a race there. Bleh, was trying to avoid the error string comparison :(.